development/server/flume/README.md

```diff
@@ -0,0 +1,35 @@
+
+
+# Apache Flume
+Apache Flume is a distributed, available and reliable tool to efficiently collect, aggregate and move large amounts of log data. Apache Flume has a simple and flexible architecture based on message flows. This tool has fault tolerant, failover and recovery mechanisms.
+
+## Prerequisites
+To use Apache Flume, you need to initialize the Kubernetes cluster and deploy on it all components of server folder of this repository following the instructions present in README file of that folder.
+
+## Installation
+In this folder, there are several files related to Apache Flume in order to deploy it on Kubernetes Cluster.
+The first step of that deploy is to build an Apache Flume image in Kubernetes Worker node. To do that, execute the following command.
+```
+sh build.sh
+```
+All commands bellow are fully automated in setup-testing-mode.sh script presents in server folder of this repository. The purpose of this section is only explain the reason of executing each command.
```

Comment:
- typo: bellow -> below
- setup-testing-mode.sh -> replace to `setup-testing-mode.sh`
- presents -> presented
- "explain" -> "to explain"
- reason -> "reasons"